### DISCUSSION TITLE
Using `\DocumentMetadata` in `main.tex`

### DISCUSSION THREAD

**Discussion by hochleitner** [Digital-Media/HagenbergThesis#168] [Ideas]
Recently, pdflatex began showing me the following warning/problem at the beginning of `main.tex`:

```console
LaTeX DocumentMetadata: The language has not been set in
(DocumentMetadata)	\DocumentMetadata.
(DocumentMetadata)	Setting it to 'de' as fallback.
```

I was unaware of that macro, so I looked it up - https://tex.stackexchange.com/questions/686898/what-is-documentmetadata-what-key-value-pairs-does-it-take-and-when-should-i-u gives a good explanation.

We should keep this in the back of our heads since it seems vital for creating tagged (accessible) PDFs in the future. The only issue I see is that this has to be set as the first command in a document, meaning that the document language would have to be put in two places. It is still in development, so let's watch it and see how to incorporate it.

**Comment by imagingbook** [comment]
As far as MikTeX is concerned this feature must have been added very recently (it was the only addition when I updated today). The requirement for being the first command is a certain inconvenience, but it will be the only way to eliminate the warnings. I suggest to add this to all documents.

Then again, as you say, the use of "de" _and_ "german", respectively, is anything but elegant. I wonder if we should allow ISO codes ('de, en') for the document language option too. (Would have to be mapped it to 'german/english' before being passed to other packages.)

**Comment by imagingbook** [comment]
This topic was moved to _Issues_ (https://github.com/Digital-Media/HagenbergThesis/issues/169).
